src/db-manifest.js

```diff
@@ -3,11 +3,14 @@ const io = require('orbit-db-io')
 
 // Creates a DB manifest file and saves it in IPFS
 const createDBManifest = async (ipfs, name, type, accessControllerAddress, options) => {
-  const manifest = {
+  const manifest = Object.assign({
     name: name,
     type: type,
     accessController: path.join('/ipfs', accessControllerAddress)
-  }
+  },
+  // meta field is only added to manifest if options.meta is defined
+  options.meta !== undefined ? { meta: options.meta } : {}
```

Comment:
What does it write if you simply set options.meta to null? Does it not just omit the field?

Reply:
Yeah I've encountered this omit behaviour a couple of times with the http-api, and I consider it to be undesirable in most cases because I don't wish to check if the intermediate object exists when using child properties. e.g.: 
`const abc = options.meta.something`  turn into
`const abc = options.meta && options.meta.something`
having a guarantee that an intermediate object exists even if it's empty is the better choice IMHO
I realise however that I have no say in the matter because it's not even my PR :P 
I guess the flip-side is trying to keep the manifest as lightweight as possible.
It's a trade-off like all things I guess.

Reply:
@aphelionz im assuming you mean to have the meta property set to options.meta in the manifest object and options.meta set to null if undefined beforehand? This defines the meta field as null in the object and is added to ipfs which changes the db address. The goal of this part of the code is to only add the meta field to the manifest if defined so creating dbs with the same config as before gives the same address.

Reply:
`options.meta = Object.assign({}, options.meta)`

Reply:
@phillmac thats fine but if options.meta is not an object there may be a problem. also how are you adding that to the manifest object?

Reply:
My bad I didn't quite understand what you were doing there

Reply:
I think it would be best if meta field is only added to the manifest if explicitly defined in options, primarily because of db addresses. if there is no meta field defined its defined as undefined in db.options.meta anyway

Reply:
I agree, your way is backwards compatible. Just checking!

Reply:
I think readability could be improved if this was compacted a little to `options.meta ? { meta: options.meta } : {}`

Reply:
my thought for using  `options.meta !== undefined` here was users being able to store any value they wanted including falsey values but thats not a big deal, ik i will almost only be using it to store objects.

Reply:
Fair point! I think in that case what you have here makes sense